src/http/request.rs

```diff
@@ -94,11 +104,21 @@ impl<'js> Request<'js> {
             None
         };
 
+        let signal = if let Some(signal) = &self.signal {
+            Some(Class::<AbortSignal>::instance(
+                ctx.clone(),
+                signal.borrow().clone(),
```

Comment:
We should not clone signal value but rather clone the signal reference no? If we abort the first request the cloned request should be aborted as well.
`signal.clone()`